server/index.ts

```diff
@@ -1,17 +1,13 @@
 import path from 'path';
 import express from 'express';
-import axios from 'axios';
 
-import eventListingsRouter from './routes/eventListingsRouter';
-import artistsRouter from './routes/artists';
+import router from './routes/index';
 
 const app = express();
 
 app.use(express.static(path.join(__dirname, '../public')));
 
-//ROUTERS------------------------------
-app.use('/events', eventListingsRouter);
-app.use('/artists', artistsRouter);
+router(app);
```

Comment:
as discussed during legacy, this is not the appropriate way to use express router. do not refer to your legacy project code for this functionality